profiles/migrations/0034_data_migration_for_countries.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-04-06 13:19
+from __future__ import unicode_literals
+import csv
+import os
+from django.db import migrations
+
+from profiles.models import Country
+
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+def forwards(apps, schema_editor):
+    """load list of countries from file"""
+    countries_file = 'data/countries.csv'
+    countries_file_path = os.path.join(os.sep, BASE_DIR, countries_file)
```

Comment:
Can you remove `os.sep` here? It shouldn't be necessary, `BASE_DIR` is an absolute path by itself